ibm/provider/provider.go

```diff
@@ -750,6 +750,7 @@ func Provider() *schema.Provider {
 			"ibm_cis_tls_settings":                      cis.ResourceIBMCISTLSSettings(),
 			"ibm_cis_waf_package":                       cis.ResourceIBMCISWAFPackage(),
 			"ibm_cis_webhook":                           cis.ResourceIBMCISWebhooks(),
+			"ibm_cis_orig_auth":                         cis.ResourceIBMCISOriginAuthPull(),
```

Comment:
spelling mistake ibm_cis_orig_auth

Reply:
OK. It was intentional but I'll change it to origin.